### DISCUSSION TITLE
How should i add a pulse excitation

### DISCUSSION THREAD

**Discussion by whtwu** [lululxvi/deepxde#828] [Q&A]
Dear Dr. Lu Lu and friends：
 I have a beam, the length is 1 meter, I want to add a pulse excitation at the position of 0.2 meters of the beam, the amplitude of the pulse excitation is 1000 N, the addition time is 0.001 seconds, and put it as the equation initial conditions, how should the code of dde.IC be implemented? thanks for your help.
![image](https://user-images.githubusercontent.com/67300153/181880018-2bf44d0f-618d-445a-906b-3304eafe09bb.png)

**Comment by lululxvi** [comment]
https://github.com/lululxvi/deepxde/issues/200#issuecomment-774400970
